src/parser/shaman/elemental/modules/azerite/EchoOfTheElementals.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer, {SELECTED_PLAYER, SELECTED_PLAYER_PET} from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import { formatNumber } from 'common/format';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+
+class EchoOfTheElementals extends Analyzer {
+  procs = 0;
+  damageGained = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrait(SPELLS.ECHO_OF_THE_ELEMENTALS.id);
+
+    let summonSpell = SPELLS.EMBER_ELEMENTAL_SUMMON;
+    let damageSpells = [SPELLS.EMBER_BLAST];
+    if (this.selectedCombatant.hasTalent(SPELLS.STORM_ELEMENTAL_TALENT.id)) {
+      summonSpell = SPELLS.SPARK_ELEMENTAL_SUMMON;
+      damageSpells = [SPELLS.SHOCKING_BLAST];
+    }
+    this.addEventListener(Events.summon.by(SELECTED_PLAYER).spell(summonSpell), this.onSummon);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER_PET).spell(damageSpells), this.onPetDamage);
+  }
+
+
+  onSummon(event) {
+    this.procs += 1;
+  }
+
+  onPetDamage(event) {
+    this.damageGained += event.amount;
+  }
+
+  statistic() {
+    return (
+      <TraitStatisticBox
+        position={STATISTIC_ORDER.OPTIONAL()}
+        trait={SPELLS.ECHO_OF_THE_ELEMENTALS.id}
+        value={`${formatNumber(this.damageGained)} damage`}
```

Comment:
Maybe replace this with 
`value={<ItemDamageDone amount={this.damageGained} />}` 
See how you think it looks, I think a lot of traits are using it, but if you prefer your current look it wont block the PR